.travis.yml

```diff
@@ -38,7 +38,7 @@ install:
   - conda config --add channels conda-forge;
   - conda config --set always_yes yes --set changeps1 no
   - if [ "${PYVER}" = "2.7" ] || [ "${PYVER}" = "3.6" ] || [ "${PYVER}" = "3.7" ]; then
-      conda create -q -n testenv python=${PYVER} root xrootd;
+      conda create -q -n testenv python=${PYVER} root;   # xrootd (to enable XRootD test in issue240)
```

Comment:
This won't change anything as `xrootd` is a dependency of `root` so it will be installed regardless. I haven't yet made time to create a [`root-minimal`](https://github.com/conda-forge/root-feedstock/issues/7) package.

> But I think I'm going to remove that test from production: installing prerequisites multiplies testing time by 10×.)

We could move to using containers which have the prerequires for each test in the matrix already installed?

Reply:
Switching to containers could help a lot—something to consider. This process of debugging the XRootD update through Travis was a slow but necessary step, but it turns 2 minute tests into 20 minute tests, where all of the extra time is spent in the conda install.

If `xrootd` is a dependency of `root`, then removing it shouldn't affect whether the test runs, and I'll see `test_issue240` in the logs. However, that doesn't explain why I'm seeing such a long install time.

Reply:
It seems [Python 2 is 50% slower](https://travis-ci.org/scikit-hep/uproot/builds/501888438) than Python 3.6 and 3.7 (the others don't install ROOT). On the conda side it's being worked but it's still annoying.

I have a few ideas. I'll play around with it on my fork over the next day or two and come up with something that is faster.

Reply:
Oh, dropping `xrootd` doesn't do anything to testing time, so it really is being pulled in by `root`. And the long times might be caused by the `root` installation. Apparently, I haven't been paying attention to how long these have been taking.

I know about the timing difference because it's not required for everything in the test matrix, not because I remember it being shorter before this PR.

Reply:
Also, `test_issue240` is still running, so I guess I'm not giving up the test. The installation time to enable ROOT-writing tests gives me the XRootD test for free.

Preparing containers could improve the long times that bothered me in this test (because I couldn't test locally), but then we'd be responsible for maintaining those containers, right? I'm not sure that's a good trade-off. I've also been hearing that the Azure tests have been running much faster. For the time being, this is fine, but I guess we can think about it.